src/Microsoft.DotNet.Cli.Utils/RuntimeConfig.cs

```diff
@@ -32,8 +33,10 @@ public static bool IsApplicationPortable(string entryAssemblyPath)
         } 
  
         private JObject OpenRuntimeConfig(string runtimeConfigPath) 
-        { 
-            return JObject.Parse(File.ReadAllText(runtimeConfigPath)); 
+        {
+            var reader = new JsonTextReader(new StringReader(File.ReadAllText(runtimeConfigPath)));
```

Comment:
I'm surprised this JObject.Parse does not implement it this way if this is faster